src/templates/json-editor.html

```
@@ -45,7 +45,7 @@ <h3 data-action='toggle-next' data-opened-icon='fa-caret-down' data-closed-icon=
     </div>
     <hr>
 
-    <button class='btn lh-normal border-0 px-0' type='button' data-action='toggle-next' aria-expanded='false' title='{{ 'Information'|trans }}' aria-label='{{ 'Information'|trans }}'><i class='fas fa-fw fa-info-circle'></i></button><span hidden><a target='_blank' class='external-link' rel='noopener' href='https://doc.elabftw.net/docs/usage/metadata'>{{ 'See documentation about extra fields.'|trans }}</a></span>
+    <button class='btn lh-normal border-0 px-0' type='button' data-action='toggle-next' aria-expanded='false' title='{{ 'Information'|trans }}' aria-label='{{ 'Information'|trans }}'><i class='fas fa-fw fa-info-circle'></i></button><span hidden><a target='_blank' class='external-link' rel='noopener' href='https://doc.elabftw.net/docs/usage/metadata'>{{ 'See documentation about custom fields.'|trans }}</a></span>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)